lib/remarkable_ruby/objects/folder.rb

```diff
@@ -1,14 +1,15 @@
 module RemarkableRuby
-  class Folder < OpenStruct
-    def initialize(attributes)
-      attributes.merge!(Folder.defaults) if attributes.keys.count <= 2
+  class Folder < Object
+    def initialize(attrs, connection)
+      attrs.merge!(Folder.defaults) if attrs.keys.count <= 2
```

Comment:
Can we add a comment here to explain why we're doing this comparison? And maybe name the "2" as a constant so it doesn't feel like a magic number.

Reply:
Yea I agree with you. I made this a little cleaner in the other pull request I made (#12).